### PR TITLE
Adding Powershell Startup and Teardown

### PR DESCRIPTION
**Pull request by fifthecho**

### CODE REVIEW

File: deployments/quick-start/setup.ps1 [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+# Set up URLs:
+vault write intermediate-ca/config/urls issuing_certificates="http://vault:8200/v1/intermediate-ca/ca" crl_distribution_points="http://vault:8200/v1/intermediate-ca/crl"
+
+# Enable the AppRole backend
+vault auth-enable approle
+
+# Create a role to allow Kubernetes-Vault to generate certificates:
+vault write intermediate-ca/roles/kubernetes-vault allow_any_name=true max_ttl="24h"
+
+# Inspect the policy file deployments/quick-start/policy-kubernetes-vault.hcl
```
Comment (F21) [inline deployments/quick-start/setup.ps1:122]: This comment can be removed.

File: deployments/quick-start/setup.ps1 [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+# Deploy
+kubectl apply -f $SAMPLE_APP_DEPLOYMENT
+
+# 4. Confirm that each pod of the sample app received a Vault token
+Write-Host "`n`nView the logs using the Kubernetes dashboard or kubectl logs mypod
+and confirm that each pod receive a token. The token and various other
+information related to the token should be logged.`n"
+
+
+# Stop-Process $proxy
```
Comment (F21) [inline deployments/quick-start/setup.ps1]: This appears to be commented out, so can it be removed?

**Review by F21 — COMMENTED** [review]
Thanks for your PR! 👍 Just a few comments! Can you please update and squash?

I am not too familiar with Powershell, but it looks great from what I can see. Can you also confirm that you've tested it and it works correctly?

File: deployments/quick-start/setup.ps1 (hunk 1 above)
Comment (fifthecho) [reply deployments/quick-start/setup.ps1:122]: That's copied directly from the setup.sh

**Comment by fifthecho** [comment]
Sorry. I mucked up the squash. Closing and will re-open momentarily.

**Pull request closed without merging**